api/lib/validation/validationMiddleware.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+var logger = require('../log/logger');
+var _ = require('underscore');
+var schemaStructureValidator = require('./schemaValidator');
+var attributeValidator = require('./attributeValidator');
+module.exports = function(req, res, next) {  
+  schemaStructureValidator.validatePolicy(req.body, function(validationResult) {
+    if(!_.isEmpty(validationResult)) {
+      logger.error('Input policy json schema is not valid', validationResult);
```

Comment:
Need more contextual info so we can correlate it to the request, like the app id?